src/crimson/os/seastore/journal.h

```diff
@@ -34,7 +34,7 @@ class Journal {
     crimson::ct_error::input_output_error
     >;
   using open_for_write_ret = open_for_write_ertr::future<journal_seq_t>;
-  virtual open_for_write_ret open_for_write() = 0;
+  virtual open_for_write_ret open_for_write(bool is_mkfs) = 0;
```

Comment:
If we need to add some work during mkfs, probably, creating mkfs() interface in Journal is a better way than adding a flag to indicate open_for write is called during mkfs?

Reply:
Will introduce `open_for_mount()` and `open_for_mkfs()` for Journal.

Reply:
done.